api/rpc/server.go

```diff
@@ -0,0 +1,60 @@
+package rpc
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/filecoin-project/go-jsonrpc"
+)
+
+type Server struct {
+	srv      *http.Server
+	handler  *jsonrpc.RPCServer
```

Comment:
this naming is confusing, maybe lets do 

`http` and `rpc` ?

Reply:
done